lib/jsonapi_compliable/errors.rb

```diff
@@ -44,5 +44,8 @@ def pretty(input)
         end
       end
     end
+
+    class RecordNotFound < StandardError
```

Comment:
BTW, something I just found via twitter on the [ruby `Exception` documentation](https://ruby-doc.org/core-2.2.0/Exception.html):

> It is recommended that a library should have one subclass of StandardError or RuntimeError and have specific exception types inherit from it. This allows the user to rescue a generic exception type to catch all exceptions the library may raise even if future versions of the library add new exception subclasses.